plugin/trino-ignite/pom.xml

```diff
@@ -0,0 +1,179 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.trino</groupId>
+        <artifactId>trino-root</artifactId>
+        <version>359-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>trino-ignite</artifactId>
+    <description>Trino - Ignite Connector</description>
+    <packaging>trino-plugin</packaging>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <ignite.version>2.10.0</ignite.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-base-jdbc</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-matching</artifactId>
+            <scope>compile</scope>
```

Comment:
This is default scope. Please remove.

Reply:
Get this error build from my local after remove this dep:

[INFO] --- maven-dependency-plugin:3.1.2:analyze-only (default) @ trino-ignite ---
[WARNING] Used undeclared dependencies found:
[WARNING]    io.trino:trino-matching:jar:359-SNAPSHOT:compile

Reply:
same below, remove all `<scope>compile</scope>` lines